### PR TITLE
Marketplace beta

### PR DESCRIPTION
**Pull request by Kryan92**

### CODE REVIEW

File: _assessment-reports/beta-digital-marketplace.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+layout: assessment-report
+title: Digital Marketplace 			
+breadcrumb: Digital Marketplace		
+permalink:	/standard/assessments/digital-marketplace-beta/
+report-excerpt: The Digital Marketplace simplifies the process of government procurement and makes it easier for businesses of all sizes to access government contracts.
+department: Digital Transformation Agency			
+date: 2017-03-15
+assessment-type: Digital Transformation Agency-led
+assessment-stage: Beta
+result: Not Pass
```
Comment (bjgordon) [inline _assessment-reports/beta-digital-marketplace.md]: Should this be a lower case p?

File: _assessment-reports/beta-digital-marketplace.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+
+The team has demonstrated they can successfully operate and improve the product while membership of the team has been changing.
+
+There are a number of examples of team members performing multiple roles. This has become increasingly necessary, and is beneficial. The team are now operating as a business, with multiple team members performing the roles of data analyst and support analyst.
+
+The team doesn’t have any capability gaps, but they know they need to increase capacity. They are recruiting for additional development and service design/research team members.
+
+The team is autonomous and feel empowered to make decisions based on the information at hand. 
+
+### Criterion 3: Agile and user-centered process
```
Comment (bjgordon) [inline _assessment-reports/beta-digital-marketplace.md]: I think this should be spelt user-centred?  That's how it's spelt here https://www.dta.gov.au/standard/3-agile-and-user-centred/

File: _assessment-reports/beta-digital-marketplace.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+
+The team completed their transition to cloud.gov.au following Alpha stage and now run their product on this platform. They have seen real improvements in the speed and simplicity of updates to their product.
+
+Using open-source code from the UK’s Government Digital Service (GDS) has helped the Digital Marketplace to get up and running quickly, getting to market within 5 weeks and meeting public commitments. 
+
+Taking code from others has had some impacts on architecture, testing and accessibility. For example, all time codes were hard coded to London time. This needed modification to support Australia’s multiple  time zones.
+
+Other impacts included:
+
+**Architecture**
```
Comment (bjgordon) [inline _assessment-reports/beta-digital-marketplace.md]: Add a line break

File: _assessment-reports/beta-digital-marketplace.md [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+The GDS open source code is hosted across many GitHub repositories. This isn't wrong or bad, but it does mean the team need to touch several repositories to make a small change.
+
+**Testing**
```
Comment (bjgordon) [inline _assessment-reports/beta-digital-marketplace.md]: Add a line break

File: _assessment-reports/beta-digital-marketplace.md [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+The team worked to integrate the existing GDS tests into their testing strategy. This has taken more time than expected but the whole team see the value in doing it.
+
+**Accessibility**
```
Comment (bjgordon) [inline _assessment-reports/beta-digital-marketplace.md]: Add a line break

File: _assessment-reports/beta-digital-marketplace.md [hunk 6]
```diff
@@ -0,0 +1,207 @@
...
+
+### Criterion 10: Test the service
+
+A load test was conducted prior to the announcement of the Digital Marketplace in August, but there are no plans for additional load testing. The team are investigating the slowest running journeys to ensure performance under load is adequate.
+
+The team should also include automated accessibility testing as part of their routine testing process.
+
+## Follow the service
+
+Visit the [Digital Marketplace](https://marketplace.service.gov.au/)
```
Comment (bjgordon) [inline _assessment-reports/beta-digital-marketplace.md]: Add a full stop

File: _assessment-reports/beta-digital-marketplace.md [hunk 7]
```diff
@@ -0,0 +1,207 @@
...
+
+### Code
+
+This service’s code is hosted in open source GitHub/x repository/ies:
```
Comment (bjgordon) [inline _assessment-reports/beta-digital-marketplace.md]: Is tis right?

File: _assessment-reports/beta-digital-marketplace.md (hunk 1 above)
Comment (Kryan92) [inline _assessment-reports/beta-digital-marketplace.md]: Done

File: _assessment-reports/beta-digital-marketplace.md (hunk 2 above)
Comment (Kryan92) [inline _assessment-reports/beta-digital-marketplace.md]: Done

**Pull request merged**